gateway/src/test/java/io/zeebe/gateway/api/topology/TopologyTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.gateway.api.topology;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.zeebe.gateway.api.util.GatewayTest;
+import io.zeebe.gateway.impl.broker.cluster.BrokerClusterStateImpl;
+import io.zeebe.gateway.protocol.GatewayOuterClass.BrokerInfo;
+import io.zeebe.gateway.protocol.GatewayOuterClass.Partition.PartitionBrokerHealth;
+import io.zeebe.gateway.protocol.GatewayOuterClass.TopologyRequest;
+import io.zeebe.gateway.protocol.GatewayOuterClass.TopologyResponse;
+import org.junit.Test;
+
+public class TopologyTest extends GatewayTest {
+
+  @Test
+  public void shouldResponseWithInitialUnhealthyPartitions() {
+    // when
+    final TopologyResponse topologyResponse = client.topology(TopologyRequest.newBuilder().build());
+
+    // then
+    assertThat(topologyResponse.getBrokersList())
+        .isNotEmpty()
+        .allSatisfy(
+            brokerInfo ->
+                assertThat(brokerInfo.getPartitionsList())
+                    .isNotEmpty()
+                    .allSatisfy(
+                        partitionPerBroker ->
+                            assertThat(partitionPerBroker.getHealth())
+                                .isEqualTo(PartitionBrokerHealth.UNHEALTHY)));
+  }
+
+  @Test
+  public void shouldUpdatePartitionHealthHealthy() {
+    // given
+    final BrokerClusterStateImpl topology =
+        (BrokerClusterStateImpl) brokerClient.getTopologyManager().getTopology();
+    topology.setPartitionHealthy(0, 1);
+
+    // when
+    final TopologyResponse topologyResponse = client.topology(TopologyRequest.newBuilder().build());
```

Comment:
nitpick: I suggest you change the name of the variable to `response` (its clear where it comes from) and change the type definition to `var` because it can easily be inferred.

Reply:
nitpick: There's some other places where you reduce the variable name length and where you can apply `var` instead of an explicit type.